### ISSUE TITLE
unserialization inspection does not recognize allowed classes

### ISSUE THREAD

**Issue by mitelg** [kalessil/phpinspectionsea#946]
Message: `Please specify classes allowed for unserialization in 2nd argument.`
Example Code:
```php
return unserialize($this->cache->getItem(self::CACHE_ID)->get(), [Token::class]);
```
the `Token` class is imported with a use statement. but changing it to:
```php
return unserialize($this->cache->getItem(self::CACHE_ID)->get(), ['Foo\Bar\Token']);
```
makes no difference. 

PhpStorm 2018.1.5
Php Inspections (EA Ultimate) 2.0.5

**Comment by kalessil** [comment]
Looks like a bug, I'll try to reproduce and fix it.

**Comment by kalessil** [comment]
Fixed!

**Comment by JimChenWYU** [comment]
@kalessil  Is it sure to fix this bug? My version is 4.0.6.3, still report this error

![image](https://user-images.githubusercontent.com/12371374/120259214-1cf40780-c2c6-11eb-87a3-a2c32202b97b.png)
![image](https://user-images.githubusercontent.com/12371374/120259186-0ea5eb80-c2c6-11eb-8b2e-1a2f84d972fa.png)

**Comment by kalessil** [comment]
@JimChenWYU : yes. By default, all classes allowed (https://3v4l.org/E6VjF) and the inspection complains about specifying if classes allowed and which classes are whitelisted.
